README.md

```diff
@@ -54,12 +54,14 @@
   :raised_hands: <a href="https://opencollective.com/asyncapi">Become a sponsor</a> :raised_hands:
 </p>
 
-## The specification
+## Read the specification
```

Comment:
specification working document should also be listed here in the readme, I mean the `asyncapi.md` that is on root on master (after merge) to make it clear where is the current not stable version

Reply:
Done!